File: java-unit-tests/src/main/java/com/example/third_module/Item.java
package com.example.third_module;

public class Item {
    String name;
    private int quantity;

    public Item(String name) {
        this(name, 1);
    }

    public Item(String name, int quantity) {
        this.name = name;
        setQuantity(quantity);
    }

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        if (!(quantity > 0)) {
            throw new IllegalArgumentException("Quantity must be positive value.");
        }
        this.quantity = quantity;
    }

    public String toJson() {
        return "asd";
    }
}

File: python-unit-tests/my_package/third_module/item.py
from dataclasses import dataclass


@dataclass
class Item:

    name: str
    quantity: int

    def __init__(self, name: str, quantity: int = 1) -> None:
        if not quantity > 0:
            msg = "Quantity must be positive value."
            raise ValueError(msg)

        self.name = name
        self.quantity = quantity
